app/parts/02_prompts.ls

```diff
@@ -14,6 +14,11 @@ module.exports =
         message: 'Project directory'
         default: @options.dir or '.'
 
+      unless @options.scaffold_as => prompts.push do
+        name: 'scaffold'
```

Comment:
should we call that option `language`? If I understand correctly what this does :)